public/locales/hi.json

```diff
@@ -2,7 +2,7 @@
   "loginPage": {
     "title": "तलवा व्यवस्थापक",
     "fromPalisadoes": "पलिसाडो के स्वयंसेवकों द्वारा एक खुला स्रोत अनुप्रयोग",
-    "talawa_portal": "तलावा पोर्टल",
+    "talawa_portal": "तरावा प्रशासन पोर्टल",
```

Comment:
Please change it to ```तलावा प्रशासन पोर्टल``` instead of ```तरावा प्रशासन पोर्टल```

Reply:
@anshgoyalevil I made the change in hi.json file as required